### ISSUE TITLE
Fix typo: Attendence --> Attendance

### ISSUE THREAD

**Issue by garg3133** [garg3133/JagratiWebApp#21]
**Details on the issue:**
There's a typo in the complete application where `attendence` is being used instead of `attendance`. We need to fix that for the complete application.

**Places where it is to be fixed:**
- [x] README.md (beginner-friendly) [Done by @Godzilla5111]
- [x] Comments all over the application (beginner-friendly) [Done by @96RadhikaJadhav]
- [x] URLs and corresponding function names in views. (beginner-friendly) [Done by @96RadhikaJadhav]
- [x] Models, views and all other places. (needs some experience in Django) [Done by @GunjanDhanuka]

**Note:** This issue can be assigned to multiple people simultaneously who can work on removing the typo at different places mentioned above.

For any further doubts and queries, feel free to join the community and ask there: https://discord.gg/Ek9q45ZjAv

**Comment by Godzilla5111** [comment]
@garg3133 please assign this to me, I wish to contribute!

**Comment by garg3133** [comment]
Sure @Godzilla5111, assigning you the README.md part.

**Comment by Godzilla5111** [comment]
Thanks @garg3133 , will fix it as the coding period begins😊. Thanks!

**Comment by 96RadhikaJadhav** [comment]
@garg3133 Could you please assign this issue to me?

**Comment by garg3133** [comment]
@96RadhikaJadhav Sure! On which part you wish to work?

**Comment by 96RadhikaJadhav** [comment]
I would like to work on Comments all over the application.

**Comment by garg3133** [comment]
@96RadhikaJadhav Assigned you the comments part.

**Comment by 96RadhikaJadhav** [comment]
@garg3133  I have created a PR. Please review and let me know if any changes required :) . Once this is done, I would like to take up next issue, which is URLs and corresponding function names in views.

**Comment by garg3133** [comment]
You can go ahead with the next issue, @96RadhikaJadhav.

**Comment by 96RadhikaJadhav** [comment]
@garg3133  Should I create another PR or should commit in the existing one?

**Comment by garg3133** [comment]
@96RadhikaJadhav You should create an another PR. Remember to pull the changes first.

**Comment by 96RadhikaJadhav** [comment]
@garg3133 Got stuck and seeking help. When I created a new branch to create another PR, I could see the files are not reflecting the changes which I did previously while creating PR  which is merged now. When I'm switching to the previous branch where changes are done, it's reflecting the changes in files. So I didn't get when to run this command-> git pull origin master. Whether to switch on previously created branch and run git pull command and have to create a PR or to create new branch and run git pull command and then create a PR.

**Comment by garg3133** [comment]
Yes, the changes you made in the previous branch won't reflect in your new branch, that's the whole concept of branches, right?

You need to create a new branch and the run `git pull upstream master` (not `origin master` as origin points to your forked repository while upstream refers to the original repository). Make sure you've set the `upstream` to point to the original repository first by running `git remote add upstream https://github.com/garg3133/JagratiWebApp`.

Once you've pulled the previous changes, you can make changes for the next part of the issue and create a new PR :)

**Comment by 96RadhikaJadhav** [comment]
@garg3133  Thank you so much for the explanation. I have created a PR. Please review and let me know if any changes required.

**Comment by 96RadhikaJadhav** [comment]
@garg3133  Done requested changes. Could you please review and let me know if any other changes required?

**Comment by GunjanDhanuka** [comment]
I can fix the typo in the Django app part. Please assign it to me!

**Comment by garg3133** [comment]
Assigned it to you @GunjanDhanuka!

**Comment by garg3133** [comment]
Resolved!!
